diktat-gradle-plugin/pom.xml

```diff
@@ -161,7 +161,7 @@
                 </os>
             </activation>
             <properties>
-                <skip.gradle.build>true</skip.gradle.build>
+                <gradle.executable>${project.basedir}/gradlew.bat</gradle.executable>
```

Comment:
why .bat
what to do in case non-Windows systems?

Reply:
By default this property is `${project.basedir}/gradlew`, separate profile is for windows because executable path is different.